.env.example

```
@@ -1,4 +1,8 @@
 DATABASE_URL=postgresql://postgres:password@localhost:5432/unstmsg
 NODE_ENV=development
 
-NEXT_PUBLIC_APP_URL=http://localhost:3000
\ No newline at end of file
+NEXT_PUBLIC_APP_URL=http://localhost:3000
+
+SPOTIFY_CLIENT_ID=
+SPOTIFY_CLIENT_SECRET=
+SPOTIFY_REFRESH_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)